docs/source/usage/plugins/phaseSpace.rst

```diff
@@ -111,7 +111,7 @@ Known Limitations
 ^^^^^^^^^^^^^^^^^
 
 - only one range per selected space-momentum-pair possible right now (naming collisions)
-- charge deposition uses the counter shape for now (would need one more write to neighbours to get it correct to the shape)
+- charge deposition uses the counter shape for now (would need one more write to neighbors to get it correct to the shape)
```

Comment:
possibly rephrase: *to evaluate it correctly according to the shape*

Reply:
fixed